alchemist-loading/src/test/java/it/unibo/alchemist/test/TestYAMLLoader.java

```diff
@@ -162,6 +162,11 @@ void testDependencies() {
         assertEquals(dependencies.get(0), "dependencies_test.txt");
     }
 
+    @Test
+    void testMaxAliases() {
+        testNoVar("yamlAliases/aliases.yml");
```

Comment:
does this check that all the expected nodes get created?

Reply:
No, it checks that the simulation is correctly loaded and run.

Reply:
There should be a quick way to get back the environment (or to pass a lambda that can operate on it), we should check that the environment is not empty.